example/lstm_master.py

```diff
@@ -1,6 +1,9 @@
 import numpy
 import time
 
+import sys
+sys.path.append('..')
```

Comment:
All this is inside the platoon packages. Why this is used if not for the "import channel" bellow?